core/controllers/base.py

```diff
@@ -367,7 +367,7 @@ def validate_and_normalize_args(self):
                 # 'csrf_token' has been already validated in the
                 # dispatch method.
                 continue
```

Comment:
I think, for this one, maybe use `pass` and keep the elif? That way the treatment of all the different parts is uniform.

Reply:
Done!